_readme/README.jl

```diff
@@ -4,63 +4,130 @@
+# Gridap.jl simulations. This package is thought as a built-in tool to assess the user in their FE calculations with a smoother workflow 
+# in a highly intuitive API.
 #
-# # This package is experimental
-#
-# Things can and will break without deprecation or warning
+# ## Installation
+
+# According to Makie's guidelines, it is enough to install one of its backends, e.g. GLMakie. Additionally, Gridap provides the plot objects
+# to be visualized
+
+# ```julia
+# julia> ] 
+# julia> add Gridap, GridapMakie, GLMakie
+# ```
 #
-# # Examples
-using Gridap, GridapMakie, Makie
+# ## Examples
 
-model = simplexify(CartesianDiscreteModel((-2pi,2pi,-pi,pi), (20,10)))
-scene = plot(model)
-save("images/2d.png", scene) #!md
-# ![](_readme/images/2d.png)
+# First things first, we shall be using the three packages
 
-data = GridapMakie.demo_data(spacedim=1, valuetype=Float64)
-scene = plot(data.u, data.model)
+using Gridap, GridapMakie, GLMakie
 using FileIO #!md
-save("images/plot_1d_Scalar.png", scene) #!md
-# ![](_readme/images/plot_1d_Scalar.png)
-
-
-model = simplexify(CartesianDiscreteModel((-2pi,2pi,-pi,pi), (10,10)))
-V = TestFESpace(reffe=:Lagrangian, order=1, valuetype = Float64,
-    conformity=:H1, model=model)
-f(pt) = sin(pt[1])*cos(pt[2])
-u = interpolate(V, f)
-scene = mesh(u, model, color=:yellow)
-save("images/mesh_2d_Scalar.png", scene) #!md
-# ![](_readme/images/mesh_2d_Scalar.png)
-
-model = simplexify(CartesianDiscreteModel((-2pi,2pi,-pi,pi), (10,10)))
-V = TestFESpace(reffe=:Lagrangian, order=1, valuetype = Float64,
-    conformity=:H1, model=model)
-f(pt) = sin(pt[1])*cos(pt[2])
-u = interpolate(V, f)
-scene = wireframe(u, model)
-save("images/wireframe_2d_Scalar.png", scene) #!md
-# ![](_readme/images/wireframe_2d_Scalar.png)
-
-model = CartesianDiscreteModel((-1,1,-1,1), (10,10))
-V = TestFESpace(reffe=:Lagrangian, order=1, valuetype = VectorValue{2, Float64},
-    conformity=:H1, model=model)
-f(pt) = VectorValue(-pt[2], pt[1])
-u = interpolate(V, f)
-scene = arrows(u, model, arrowsize=0.1, lengthscale=0.5)
-save("images/arrows_2d_Vec2d.png", scene) #!md
-# ![](_readme/images/arrows_2d_Vec2d.png)
-
-model = CartesianDiscreteModel((-1,1,-1,1, -1, 1), (10,10, 10))
-V_pressure = TestFESpace(reffe=:Lagrangian, order=1, valuetype=Float64, conformity=:H1, model=model)
-V_velo = TestFESpace(reffe=:Lagrangian, order=1, valuetype=VectorValue{3, Float64},
-    conformity=:H1, model=model)
-
-pressure = interpolate(V_pressure, pt -> pt[1]*pt[2]*pt[3])
-velocity = interpolate(V_velo, pt -> VectorValue(-pt[3]*pt[2],pt[3]*pt[1], 0.1*pt[1]*pt[2]))
-
-color = GridapMakie.to_visualization_data(pressure, model) |> GridapMakie.get_nodalvalues
-scene = arrows(velocity, model, arrowsize=0.1, lengthscale=0.2, arrowcolor=color)
-
-save("images/arrows_3d_Vec3d_Fancy.png", scene) #!md
-# ![](_readme/images/arrows_3d_Vec3d_Fancy.png)
+
+# ### 2D Plots
+
+# Then, let us consider a simple, 2D simplexified cartesian mesh Ω
+
+domain = (0, 1, 0, 1)
```

Comment:
@fverdugo @ericneiva @paurierap ... I just realized that we are not getting Julia code highlighted syntax in the rendered `README.md` file. Is this as expected? Not sure why ... but might this be related to Literate.jl parameters?

Reply:
Solved using [Literate's interaction with Documenter](https://fredrikekre.github.io/Literate.jl/v2/documenter/). Apparently, the attribute `documenter` is `true` by default in `Literate.markdown`. Just use `Literate.markdown("README.jl", "..", documenter=false)` in `make.jl`.